src/Views/Todos.js

```diff
@@ -0,0 +1,74 @@
+import { getTodos, renderTodo, complete } from '../services/todos';
+import { useState, useEffect } from 'react';
+import './Todos.css';
+
+export default function Todos() {
+  const [list, setList] = useState([]);
+  const [error, setError] = useState('');
+  const [todo, setAddLists] = useState('');
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try { 
+        const data = await getTodos();
+        setList(data);
+      } catch (e) {
+        setError('Did not import list');
+      }
+    };
+    fetchData();
+  }, []);
+
+  const handleSubmit = async () => {
+    try { 
+      await renderTodo({ todo });
+      history.go(0);
```

Comment:
history.go(0) will reload the page each time a chore is added. To optimize your app you can update the state using prevState which will not do a fetch each time a todo is added.